mtags/src/main/scala-2/scala/meta/internal/metals/MultilineStringFormattingProvider.scala

```diff
@@ -23,19 +22,19 @@ object MultilineStringFormattingProvider {
   type StartPosition = meta.Position
   type EndPosition = meta.Position
 
-  private def hasStripMarginSuffix(
-      stringTokenIndex: Int,
-      tokens: Tokens
-  ): Boolean = {
-    var methodIndex = stringTokenIndex + 1
-    while (tokens(methodIndex).isWhiteSpaceOrComment ||
-      tokens(methodIndex).isInstanceOf[Token.Dot]) methodIndex += 1
-    tokens(methodIndex) match {
-      case token: Token.Ident if token.value == stripMargin =>
-        true
-      case _ =>
-        false
-    }
+  def hasStripMarginSuffix(token: Token): Boolean =
+    !stripMargin.zipWithIndex
+      .exists {
+        case (char, index) =>
+          token.input.chars(token.pos.end + index + 1) != char
```

Comment:
This might fail if we have a whitespace somewhere between like `"""|""" stripMargin` That's why previously we used a while loop to skip over dot or whitespace.

Reply:
I didn't know we could have more than one space after the multiline string. 
Right now it works infact only if there is a space or "." after the multiline string.